codegen/src/main/twirl/templates/JavaServer/Handler.scala.txt

```diff
@@ -21,6 +21,9 @@ import akka.grpc.ProtobufSerializer;
 import akka.grpc.javadsl.GoogleProtobufSerializer;
 import akka.grpc.javadsl.GrpcMarshalling;
 import akka.grpc.javadsl.GrpcExceptionHandler;
+import akka.grpc.javadsl.package$;
```

Comment:
I saw that `akka.japi.Function` was used consistently instead of `akka.japi.function.Function` in codegen/ and runtime/ so followed that example in using it for the exception handler. In this file we need the conversion to scala `PartialFunction` from the package object.

Reply:
Yes, but using `akka.japi.Function` over `akka.japi.function.Function` might have been a mistake, as the javadocs on the former say "This class is kept for compatibility, but for future API's please prefer [[akka.japi.function.Function]]". But we can do that in a separate PR.

Reply:
Added #513